### PR TITLE
Add ability to store a users settings

### PR DESCRIPTION
**Pull request by adgad**
/cc @commuterjoy @wheresrhys

### CODE REVIEW

**Comment by adgad** [comment]
Any preference(no pun intended) on the naming?

preferred/Setting
saved/Preference(or Setting) - although this might get confusing with saved for later
customized/Setting

File: models/Actor.js [hunk 1]
```diff
@@ -10,26 +10,15 @@ var Actor = function (id) {
...
-    return Activities.query(this.id, 'forlater.Article:');
-}
+/* Write operations */
 
-Actor.prototype.notified = function () {
-    return Activities.query(this.id, 'notified_of.Article:');
+Actor.prototype.preferred = function (id, meta) {
+    return Promise.all([
+        Activities.save(this.id, 'preferred.' + id, meta),
+        Activities.save(id, 'preferred_by.' + this.id, meta)
```
Comment (commuterjoy) [inline models/Actor.js]: Do we need the bi-directional relationship?

**Comment by commuterjoy** [comment]
@adgad 

The API was built with [subject-verb-object](http://en.wikipedia.org/wiki/Subject-verb-object) in mind.

So,

```
<User> <Preferred> <Preference:Foo>  <-- preference, not setting
<Preference:Foo> <Preferred_By> <User>

File: models/Actor.js (hunk 1 above)
Comment (adgad) [inline models/Actor.js]: My thought was, e.g if we had a scheduled job that sent a daily email to everyone who had a daily email preference set

Comment (commuterjoy) [inline models/Actor.js]: Yeah, good thought.

**Comment by commuterjoy** [comment]
sorry. hit the wrong button :)

**Pull request merged**